src/python/grpcio_tests/tests/unit/_version_test.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test for grpc.__version__"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+import grpc
+import logging
+
+
+class VersionTest(unittest.TestCase):
+
+    def test_get_version(self):
+        self.assertIsInstance(grpc.__version__, str)
```

Comment:
This would be more useful if it could compare against `VERSION` from https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc_version.py#L17